exporter/trace/cloudtrace.go

```diff
@@ -245,6 +245,14 @@ func newContextWithTimeout(ctx context.Context, timeout time.Duration) (context.
 
 // ExportSpans exports a ReadOnlySpan to Stackdriver Trace.
 func (e *Exporter) ExportSpans(ctx context.Context, spanData []sdktrace.ReadOnlySpan) error {
+	sd := make([]ReadOnlySpan, len(spanData))
+	for i := range spanData {
+		sd[i] = spanData[i]
+	}
+	return e.traceExporter.ExportSpans(ctx, sd)
+}
+
+func (e *Exporter) ExportCustomSpans(ctx context.Context, spanData []ReadOnlySpan) error {
```

Comment:
Is this new function used for anything?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4146/commits/b7807e08f0cb1f0f84f351678c9548b3240b1cc3